src/utils/poaps.ts

```diff
@@ -0,0 +1,23 @@
+import { arcClient, arcDevClient } from '@/graphql';
+import { Navigation } from '@/navigation';
+import Routes from '@/navigation/routesNames';
+import { IS_DEV } from '@/env';
+
+export type PoapMintError = 'LIMIT_EXCEEDED' | 'EVENT_EXPIRED' | 'UNKNOWN';
+
+const POAP_REGEX = /(?:[A-Za-z]+-){2}[A-Za-z]+$/;
+
+export const checkValidSecretWord = (secretWord: string) => {
+  return POAP_REGEX.test(secretWord);
+};
+
+export const getPoapAndOpenSheet = async (secretWord: string) => {
+  const client = IS_DEV ? arcDevClient : arcClient;
```

Comment:
you have two instances of `IS_DEV ? arcDevClient : arcClient;`, and i know i do it in the nft offers feature also. should we just have a universal reference to the arc client (taking into account dev vs prod env) somewhere?

Reply:
yeah this is a great idea